crates/sncast/src/starknet_commands/account/import.rs

```diff
@@ -139,3 +154,9 @@ fn get_private_key_from_file(file_path: &Utf8PathBuf) -> Result<Felt> {
     let private_key_string = std::fs::read_to_string(file_path.clone())?;
     Ok(private_key_string.parse()?)
 }
+
+fn get_private_key_from_input() -> Felt {
+    let private_key =
+        rpassword::prompt_password("Enter private key: ").expect("Failed to read private key");
+    Felt::try_from_hex_str(&private_key).unwrap()
+}
```

Comment:
This should be definetely tested somehow. I've tried it locally and it fails on invalid felt input. 

1. We should test that 
2. We should handle felt errors instead of unwrapping.

Reply:
Also this should allow integer input as well, not only hex

Reply:
I've added function for parsing and relevant tests.